projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/select-docs/select-docs.component.ts

```diff
@@ -188,6 +197,27 @@ export class SelectDocsComponent implements OnInit {
   );
   `;
 
+  select12Code: string = `
+  <go-select
+    bindLabel="name"
+    bindValue="value"
+    [control]="select"
+    groupBy="manufacturer"
```

Comment:
Assuming this has to match the *exact* property on your objects, right? Just asking for curiosity's sake.

Reply:
Yup it has to match exactly